src/components/footer/Footer.jsx

```diff
@@ -0,0 +1,68 @@
+                <List title="Over NOVI">
+                    <ResourceLinks url="https://www.test.nl" target="_blank" icon_only={false} name="Opeidingen" />
+                    <ResourceLinks url="https://www.test.nl" target="_blank" icon_only={false} name="Studeren Bij" />
+                    <ResourceLinks url="https://www.test.nl" target="_blank" icon_only={false} name="Blog" />
+                    <ResourceLinks url="https://www.test.nl" target="_blank" icon_only={false} name="Contact" />
+                    <ResourceLinks url="https://www.test.nl" target="_blank" icon_only={false} name="Vacatures"/>
+                </List>
+                <List title="Studenten">
+                    <ResourceLinks url="https://www.test.nl" target="_blank" icon_only={false} name="Kennisbank" />
+                    <ResourceLinks url="https://www.test.nl" target="_blank" icon_only={false} name="Edhub"/>
+                    <ResourceLinks url="https://www.test.nl" target="_blank" icon_only={false} name="Subsidie"/>
+                    <ResourceLinks url="https://www.test.nl" target="_blank" icon_only={false} name="Verkort HBO-traject"/>
+                </List>
+                <List title="In Company">
+                    <ResourceLinks url="https://www.test.nl" target="_blank" icon_only={false} name="Business" />
+                    <ResourceLinks url="https://www.test.nl" target="_blank" icon_only={false} name="Financiering"/>
+                </List>
+            </section>
+            <section className="footer-sub-container">
+                <List title="Contact">
+                    <ResourceLinks url="https://www.test.nl" target="_blank" icon_only={false} name="contact@novi.nl" />
+                    <ResourceLinks url="https://www.test.nl" target="_blank" icon_only={false} name="030-3073200" />
+                    <ResourceLinks url="https://www.test.nl" target="_blank" icon_only={false} name="Newtonlaan 247, Utrecht"/>
+                   <div className="footer-icon-container">
+                       <ResourceLinks url="https://www.test.nl" target="_blank" classname="footer-icon" icon_only={true} icon="https://cdn4.iconfinder.com/data/icons/ionicons/512/icon-social-twitter-outline-1024.png" />
+                       <ResourceLinks url="https://www.test.nl" target="_blank" classname="footer-icon" icon_only={true} icon="https://cdn1.iconfinder.com/data/icons/social-media-outline-6/128/SocialMedia_Instagram-Outline-1024.png" />
+                       <ResourceLinks url="https://www.test.nl" target="_blank" classname="footer-icon" icon_only={true} icon="https://cdn2.iconfinder.com/data/icons/social-media-outline-1/32/linked_in_social_media_online-1024.png" />
+                       <ResourceLinks url="https://www.test.nl" target="_blank" classname="footer-icon" icon_only={true} icon="https://cdn1.iconfinder.com/data/icons/social-media-rounded-corners/512/Rounded_Facebook_svg-1024.png"/>
```

Comment:
Hier kun je het best even [phosphoricons](https://phosphoricons.com/) voor installeren!

Reply:
En ik denk dat de social media icons niet persé een resource link zijn. De resource links wijzen naar plekken op de NOVI website, de social media icons verwijzen naar andere plekken (zoals instagram of facebook) dus ik denk dat je die beter los kunt opstellen tot we die social media icons op meer plekken gaan zien terugkomen